src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -24,7 +24,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="Runtime" />
```

Comment:
❓ Why is this changing?

Reply:
@jmarolf for this conversation as well. Since he added the ILogger interface, any use of MSBuildWorkspace APIs require you have a reference to newer versions of Microsoft.Build.Framework; this is no longer a private implementation dependency as a result.

@jmarolf has another PR out now to address the impacts of this so this might be revertible depending on how that works out.

Reply:
The approach from AnalyzerRunner can be used instead.

```
<PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
```

Reply:
If that's the reference needed then we should update MSBuildWorkspace to have just that package; requiring all downstream consumers to mess with this isn't correct.

Reply:
> requiring all downstream consumers to mess with this isn't correct.

I don't think we have a way to propagate `ExcludeAssets="Runtime"`, which is required.

Reply:
I'll be resolving this in favor of the change @jmarolf made in his PR once it merges to master.